miniapps/elasticity/makefile

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) 2010-2022, Lawrence Livermore National Security, LLC. Produced
```

Comment:
Consider adding a `README` file in this directory, see `miniapps/solvers`

(Even better, consider adding docs on the website that explain/illustrate the miniapp)